infinit/content.md

```diff
@@ -0,0 +1,45 @@
+# What is Infinit?
+
+Infinit’s storage platform transforms commodity servers into multiple flexible, scalable, secure and fault-tolerant storage logics tailored to your applications.
+
+%%LOGO%%
+
+# How to use this image?
+
+The following commands show Docker-specific tips for a pleasant integration between Infinit and Docker. For the Infinit-specific documentation, consult [Infinit's website](https://infinit.sh/get-started).
+
+## Basics
+
+	docker run --rm infinit
+
+### Sharing configuration folder
+
+Infinit stores its configuration under `/root/.local/share/infinit`. If you want to share this folder with your host, you can mount your host's configuration folder as a volume in your container.
+
+	docker run -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit
+
+*N.B. This might affect permissions of the mounted folder.*
+
+### Using your user
+
+By default, Infinit uses the current user, `root` when running on a container. If you want to use your local user name, you can set the INFINIT_USER environment variable.
```

Comment:
What does the `INFINIT_USER` environment variable do?  If I pass in my linux username, it is not going to mean anything inside the container since it does not have my `/etc/passwd` to get a user id.